### DISCUSSION TITLE
Separation logic support & splittable points-to

### DISCUSSION THREAD

**Discussion by bobismijnnaam** [cvc5/cvc5#9227] [General]
Hi, I was wondering: does CVC5 have any plans for supporting separation logic with splittable points-to predicates, or maybe fractional permissions? Or is this somehow already encodable with the current support for separation logic?

The reason I'm asking for this is that I noticed this is not discussed in papers from SL-COMP, even though I think it's an important part of a separation logic. However I can't find the mailing list, and the SL-COMP slack seems private, so I figured I'd give it a try here. Any thoughts or pointers about this question, also outside the context of CVC5, would be welcome.

If this is too off-topic for CVC5, apologies, please close this issue.

**Comment by ajreynol** [comment]
Sorry for the delay in reply.

Unfortunately we don't have support for those extensions, and there are no plans to add extend the signature of the separation logic theory at the moment.

**Reply by bobismijnnaam** [reply]
Okay, thank you!
